governance-exercises/Le Bateleur - Poll Exercise 01/Course of Action.md

```diff
@@ -0,0 +1,83 @@
+
+##### Request Faraz/Advisors confirm their agreement with increasing the Spork Deposit Rate per Alice's proposal.
+
+If polled in its current state, there would be insufficient clarity and transparency around the process that led to the poll being submitted to governance. Faraz neglected to address Alice's proposal to increase Spork's DAI deposit rate. It is necessary that the Facilitator and advisory teams confirm all proposed changes are reasonable and suitable for implementation on Spork. 
+
+##### Request confirmation of the organizing of proposals into 3 separate polls. 
+
+To ensure greater clarity and transparency around the pre-Poll process, I would confirm the parties' agreement with breaking out the proposals into three polls for the following Monday. I recommend this organization so that each Poll is focused on a distinct sub-set of the proposals.
+
+Poll 1: Spork D3M parameter adjustments
+
+Poll 2: Increase Spork DAI Deposit Rate
+
+Poll 3 (assuming Faraz/advisors provide an updated risk assessment): Onboard ABC token to Spork
+
+#### II. Prepare three (3) separate governance polls for the following Monday
+
+Assuming the parties promptly address/confirm all foregoing points, I would then post a final follow-up on the Forum to signal that the polls will proceed as set out above. 
+
+Then, I would prepare three (3) separate polls to be posted on the voting portal the following Monday, July 24. 
+
+Given the time-sensitive nature of Alice's proposal, there is a specific need to use the Weekly Governance Cycle. The governance polls are fully justified per [MIP16](https://mips.makerdao.com/mips/details/MIP16#component-summary):
+
+> The Weekly Governance Cycle is used explicitly for recurring operational decisions that require quicker action than the Monthly Governance Cycle (defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51)). For example, the Weekly Cycle will be used to modify rates and debt ceilings for the collateral types in the Maker collateral portfolio.
+>
+> It is important to note that due to the frequency of such changes, the Weekly Cycle is not suitable for long term or substantial decisions but is ideal for specific time-sensitive decisions that do not require an Emergency vote.
+>
+> ... [T]he use of the weekly cycle as defined in this MIP should only occur where there is a specific need to use the weekly cycle as opposed to the monthly cycle.
+
+Per [MIP16c3](https://mips.makerdao.com/mips/details/MIP16#MIP16c3), I assume there is consensus among the Governance Facilitators so that I may make use of the weekly cycle as opposed to the monthly cycle. 
+
+I also assume that the number of polls to be released for the following Monday is not prohibitively large. If this were the case, I might at least consider aggregating the Spork proposals into a single/smaller number of polls. (See [here](http://forum.makerdao.com/t/iam-dc-parameters-2-lrc-a-bat-a-bal-a-mana-a-zrx-a-knc-a-renbtc-a/6431/2?u=le_bateleur).) 
```

Comment:
Hah. That thread takes me back. Yes, good thing to keep an eye on. Though nowadays, there just isn't as much of a governance workload.